src/rust/engine/fs/store/src/local.rs

```diff
@@ -302,7 +302,7 @@ impl ByteStore {
 
     if let Some(workunit_store_handle) = workunit_store::get_workunit_store_handle() {
       workunit_store_handle.store.record_observation(
-        ObservationMetric::LocalStoreReadBlobSize,
+        ObservationMetric::LocalCacheReadBlobSize,
```

Comment:
This isn't necessarily cache related: the `Store` is used for local execution in general, even if the cache is disabled.

Reply:
I agree. My original "LocalStoreReadBlobSize" is more apropos.